docs/conf.py

```diff
@@ -48,16 +48,16 @@
 
 # General information about the project.
 project = u'libCellML'
-copyright = u'2014, David Nickerson'
+copyright = u'2014-2015, University of Auckland'
```

Comment:
yeah, I just made this change to bring the docs in line with the current copyright statement in the rest of the code - its not a fixed thing that is required from this end. Is there a standard way to say "copyright the contributors" ? or something similar? maybe a "Copyright libCellML Contributors" and then list contributors somewhere?